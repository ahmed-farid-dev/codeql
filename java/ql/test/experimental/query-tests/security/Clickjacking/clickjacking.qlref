experimental/Security/CWE/CWE-451/clickjacking.ql
