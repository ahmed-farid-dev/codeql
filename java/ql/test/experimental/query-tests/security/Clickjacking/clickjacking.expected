| Test.java:11:9:11:54 | setHeader(...) | Misconfigured X-Frame-Options Header. |
